typed/merge.go

```diff
@@ -180,14 +180,18 @@ func (w *mergingWalker) visitListItems(t *schema.List, lhs, rhs value.List) (err
 	}
 	out := make([]interface{}, 0, outLen)
 
-	rhsOrder, observedRHS, rhsErrs := w.indexListPathElements(t, rhs)
+	rhsPEs, observedRHS, rhsErrs := w.indexListPathElements(t, rhs)
```

Comment:
Are the PE's no longer ordered?

Reply:
They are still, but "order" didn't really tell me what they were. I could revert or make it `rhsOrderedPEs` or something.